### PR TITLE
Feature/entry update

### PR DESCRIPTION
**Pull request by TinyPandas**
- Implement stricter verification: image captcha
- Implement kick/ban buttons on join embed

### Config Changes before restart of bot
- "joinLogChannelId": ""

### CODE REVIEW

**Comment by TinyPandas** [comment]
Will be looking to fix #103 in this PR later today.

File: src/menus/captchaSelector.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+		interaction
+			.update({
+				components: [],
+				embeds: [],
+				content: "You have selected an invalid captcha.",
+			})
+			.catch(err => log(err, "error"));
+
+		if (member) {
+			//member.kick("Invalid captcha.");
```
Comment (M39a9am3R) [inline src/menus/captchaSelector.ts:62]: Intend to add 3-strike system or other implement to punish failing the captcha multiple times?

File: src/store/permissions.ts [hunk 2]
```diff
@@ -144,4 +144,14 @@ const permissions: PermissionTree = {
 			},
 		],
 	},
+	verify: {
+		field: "memberRoleId",
```
Comment (M39a9am3R) [inline src/store/permissions.ts:148]: May be wrong on this note. Would command require everyone permissions, or are we retaining everyone being granted Member roles on joining the server? Or is permission being granted to everyone on load of src/util.ts? How is this part supposed to work?

**Review by M39a9am3R — CHANGES_REQUESTED** [review]
Multiple inquiries and removed contents.

File: src/store/permissions.ts (hunk 2 above)
Comment (TinyPandas) [reply src/store/permissions.ts:148]: How this works is anyone with the verified role is no longer allowed to use the verify command.

File: src/menus/captchaSelector.ts (hunk 1 above)
Comment (TinyPandas) [reply src/menus/captchaSelector.ts:62]: This will remain commented out until we can discuss possible routes to take for multiple failed captchas.

**Review by M39a9am3R — APPROVED** [review]

**Pull request merged**